Source/items.cpp

```diff
@@ -3222,6 +3222,7 @@ void RecreateEar(Item &item, uint16_t ic, int iseed, int id, int dur, int mdur,
 	std::string itemName = fmt::format(_(/* TRANSLATORS: {:s} will be a Character Name */ "Ear of {:s}"), heroName);
 
 	CopyUtf8(item._iName, itemName, sizeof(item._iName));
+	CopyUtf8(item._iIName, heroName, sizeof(item._iIName));

```

Comment:
We should probably make LoadHeroItems() not load ears so make sure we don't get old ones where this fix has not been applied.

Reply:
Good thinking, I hadn't thought of that at all.